core/trino-main/src/test/java/io/trino/cost/TestFilterStatsCalculator.java

```diff
@@ -301,6 +311,99 @@ public void testAndStats()
 
         assertExpression("CAST(NULL AS boolean) AND CAST(NULL AS boolean)").equalTo(zeroStatistics);
         assertExpression("CAST(NULL AS boolean) AND (x < 0e0 AND x > 1e0)").equalTo(zeroStatistics);
+
+        Consumer<SymbolStatsAssertion> symbolAssertX = symbolAssert -> symbolAssert.averageRowSize(4.0)
+                .lowValue(-5.0)
+                .highValue(5.0)
+                .distinctValuesCount(20.0)
+                .nullsFraction(0.0);
+        Consumer<SymbolStatsAssertion> symbolAssertY = symbolAssert -> symbolAssert.averageRowSize(4.0)
+                .lowValue(1.0)
+                .highValue(5.0)
+                .distinctValuesCount(16.0)
+                .nullsFraction(0.0);
+
+        assertExpression(
+                "(x BETWEEN -5 AND 5) AND y > 1",
+                Session.builder(session).setSystemProperty(FILTER_CONJUNCTION_INDEPENDENCE_FACTOR, "0").build())
+                .outputRowsCount(375)
```

Comment:
What is `375`? Is it selectivity of `(x BETWEEN -5 AND 5)` or `y > 1` as I would expect so. Please add a comment